### PR TITLE
refactor: update lint and package version

### PR DESCRIPTION
**Pull request by gasolin**
and update packages

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ $ bower install gaia-components/gaia-component
 
 ```js
 var MyComponent = component.register('my-component', {
+  extends: HTMLButtonElement.prototype,
```
Comment (gasolin) [inline README.md:16]: extends from native element will inherit native element behavior and improve the accessibility

**Comment by wilsonpage** [comment]
I don't really understand the benefit of `_` prefixing methods that are concealed inside a module closure. My understanding was that `_` prefix convention is used to communicate to _users_ that methods/properties on a `prototype` are 'private' and shouldn't be used.

File: README.md [hunk 2]
```diff
@@ -12,6 +12,8 @@ $ bower install gaia-components/gaia-component
 
 ```js
 var MyComponent = component.register('my-component', {
+  // extends from parent element could inherit parent element's behavior
+  extends: HTMLButtonElement.prototype,
```
Comment (wilsonpage) [inline README.md:16]: This comment is quite vague. I think we should be clear that the only thing `extends` does is extend the given `prototype`, nothing more (unfortunately :cry:)

**Comment by gasolin** [comment]
`_` convention is widely used in system/settings to indicate this method is not called from outside of this file. It provide more visual hints than `@private`. Though its optional, I can create another PR without _prefix change

**Comment by wilsonpage** [comment]
These methods are _impossible_ to be called from outside as we use module closure. And we should never have global state floating around.

**Comment by gasolin** [comment]
removed _prefix, keep @private annotations

File: bower.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "gaia-component",
-  "version": "0.3.6",
+  "version": "0.3.7",
```
Comment (wilsonpage) [inline bower.json:3]: Remember to push a matching git tag when this lands. Usually I would stamp versions in a separate commit, tag that commit, then push straight to `master`. But this doesn't matter that much :)

Comment (wilsonpage) [inline bower.json:3]: No worries, I did it.

**Pull request merged**